app/controllers/resources_controller.rb

```diff
@@ -173,6 +193,10 @@ def blob_for_signed_id(signed_id, filename)
     raise BlobError, "Unable to find upload for #{filename} (#{signed_id})"
   end
 
+  def file_from_globus(globus_id)
+    globus_id.gsub('globus://', Settings.globus_location)
```

Comment:
If we expect just one instance of `globus://` in the string, we can consider using `String#sub` instead of `String#gsub`. (The latter suggests there may be multiple instances.)